lib/web_ui/lib/src/engine/app_bootstrap.dart

```diff
@@ -61,6 +64,14 @@ class AppBootstrap {
 
   /// Represents the App that was just started, and its JS API.
   FlutterApp _prepareFlutterApp() {
-    return FlutterApp();
+    return FlutterApp(
+      addView: (JsFlutterViewOptions options) async {
+        assert(configuration.multiViewEnabled, 'Cannot addView when multiView is not enabled');
+        return EnginePlatformDispatcher.instance.viewManager.createView(options: options);
+      },
+      removeView: (int viewId) async {
+        return EnginePlatformDispatcher.instance.viewManager.unregisterView(viewId);
+      }
```

Comment:
Do we also need `disposeView`? Can't tell how final the `removeView` operation is.

Reply:
I discussed this with @mdebbar, and this will indeed `dispose` the flutter-side of the view: we'll clear the DOM and remove listeners. (so very final, other than actually removing the hostElement from the page)